### DISCUSSION TITLE
create git repos for flux v2 and how to install flux controller in a cluster

### DISCUSSION THREAD

**Discussion by koushikyanala** [fluxcd/flux2#1305] [Q&A]
Hello all,

can anyone here tell me how  repos for flux v2 looks like and i am using AWS codecommit and if see flux check --pre iam getting error at kubernetes client initialization failed: kubernetes configuration load failed: invalid configuration: no configuration has been provided, try setting KUBERNETES_MASTER environment variable. and what is GITHUB_TOKEN for AWS code commit

**Comment by kingdonb** [comment]
AWS CodeCommit is not currently supported. If you are able to get it working, we'd love to hear more. Here are some related reports.

#808, #948
